_gsocproposals/proposal_GoHEPInterpreter.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Launching Wagon, a WebAssembly interpreter in Go
+layout: plain
+project: Go-HEP
+organization: CERN
```

Comment:
right.
should be Go-HEP, then. (I just followed the recommendation from https://github.com/HEP-SF/hep-sf.github.io/pull/83/files#r99058593)